### PR TITLE
Supporting brands taxonomy.

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

**Comment by kovacshuni** [comment]
🎄

**Review by BerniVarga — APPROVED** [review]

File: taxonomyService_test.go [hunk 1]
```diff
@@ -556,6 +589,12 @@ func buildConceptSuggestionsWithGenres(genreCount int) []suggestion {
 	return buildConceptSuggestions(taxonomyAndCount, false, false)
 }
 
+func buildConceptSuggestionsWithBrands(count int) []suggestion {
+	taxonomyAndCount := make(map[string]int)
+	taxonomyAndCount["brands"] = count
+	return buildConceptSuggestions(taxonomyAndCount, false, false)
```
Comment (ghost) [inline taxonomyService_test.go:601]: buildConceptSuggestions doesn't seem to do anything for brands?

File: taxonomyService_test.go [hunk 2]
```diff
@@ -365,6 +392,12 @@ func buildContentRefWithGenres(genreCount int) ContentRef {
 	return buildContentRef(taxonomyAndCount, false, false)
 }
 
+func buildContentRefWithBrands(count int) ContentRef {
+	taxonomyAndCount := make(map[string]int)
+	taxonomyAndCount["brands"] = count
+	return buildContentRef(taxonomyAndCount, false, false)
```
Comment (ghost) [inline taxonomyService_test.go:398]: buildContentRef doesn't appear to do anything for brands either.

File: taxonomyService_test.go (hunk 1 above)
Comment (kovacshuni) [reply taxonomyService_test.go:601]: fixed

File: taxonomyService_test.go (hunk 2 above)
Comment (kovacshuni) [reply taxonomyService_test.go:398]: fixed

**Review by ghost — APPROVED** [review]
Could you add Coveralls to the project while you're in it?  https://sites.google.com/a/ft.com/universal-publishing/how-can-i/adding-code-coverage-metrics-to-a-project

**Pull request merged**